src/actions/VaccineActions.js

```diff
@@ -38,13 +40,39 @@ const startSensorScan = () => async (dispatch, getState) => {
     return null;
   }
 
-  dispatch(scanForSensors());
+  await BleService()
+    .blinkWithRetries(macAddress, 3)
+    .catch(error => {
+      const { message } = error;
+      ToastAndroid.show(message || error, ToastAndroid.LONG);
```

Comment:
The error here could be `Parsing failed:  ${e.message}`, technically - maybe not so useful for a user?

Reply:
true, I'll rework. this one was spitballing really, as I didn't have an error. I think an invalid address will throw something, so will try that.

Reply:
Could also mess around with `data => !!stringFromBase64(data).match(/ok/i)` - essentially what's being sent back is some binary in base64 - I suppose it's technically possible for the sensor to just not send the correct response.. 🤷 

I think in general it's probably just nice to know if a blink happened or not (because I often look away and miss the actual LED blinking 🤣 )

Reply:
yeah, I think so too. the blink is pretty subtle. The error handling isn't working well for me either and I would really like it to.